apps/cli/package.json

```diff
@@ -11,6 +11,9 @@
     "node": "16"
   },
   "main": "lib/index.js",
+  "bin": {
+    "@link-to-code/cli": "./bin/cli"
```

Comment:
Should we give this a simpler name, like simply link-to-code? This is what is then used by people in terminals, isn't it?